dataactbroker/handlers/userHandler.py

```diff
@@ -58,6 +58,13 @@ def deleteToken(self,token):
         self.session.delete(oldToken)
         self.session.commit()
 
+    def getUsers(self, agency=None):
+        """ Return all users in the database """
+        query = self.session.query(User)
+        if agency is not None:
+            return query.filter(User.agency == agency)
+        return query.order_by(User.user_status_id).all()
```

Comment:
the if block should actually contain "query = query.filter(User.agency == agency"). But they code is common the only difference is adding a filter vs returning all. If we were to separate them out into 2 functions, then the only difference is that one is parameterized with a filtery_by option while the other isn't. Is that the desired way to organize these functions?